app/scripts/lib/setupSentry.js

```diff
@@ -101,6 +101,9 @@ export default function setupSentry ({ release, getState }) {
       // append app state
       if (getState) {
         const appState = getState()
+        if (!report.extra) {
+          report.extra = {}
+        }
```

Comment:
```suggestion
        report.extra = report.extra ?? {}
```

(One day we might have an [`??=`](https://github.com/tc39/proposal-logical-assignment) operator.)